components/webserver/utils/convert_html_to_header_file.sh

```diff
@@ -11,16 +11,15 @@ output_filename="page_${output_name}.h"
 echo """#ifndef PAGE_${output_name^^}_H
 #define PAGE_${output_name^^}_H
 
-const char page_${output_name,,}[] =""" > $output_filename
+// This file was generated using xxd""" > $output_filename
 
+# Gzip file and write to static array
+gzip --best "$1" -c > "page_${output_name}"
```

Comment:
do you know whether gzip is packed with Linux distributions by default? If not, there should be probably a check at the beginning with hint to install it in error message or mention in README somewhere that it's precondition

Reply:
Not overly familiar with other flavors but it is packaged in all Debian based distros by default. Also seems to be a core package in Arch.

Reply:
Ok 👍